### PR TITLE
Changed "dccf" to "CPC-vCF"

### PR DESCRIPTION
**Pull request by AroraAnshul**

### CODE REVIEW

**Comment by jim-phillips** [comment]
@AnthonyHakim Please review.
CPCVM - should be either `CPC-VCF` or `CenturyLink Private Cloud on VMware Cloud Foundation`

File: Disaster Recovery/SafeHaven 5 CPCVM to CLC/add-claim-storage-dccf-clc.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{{
-  "title": "Add and Claim Storage to SRNs-DCCF to CLC",
+  "title": "Add and Claim Storage to SRNs-CPCVM to CLC",
```
Comment (AnthonyHakim) [inline Disaster Recovery/SafeHaven 5 CPCVM to CLC/add-claim-storage-dccf-clc.md]: Change CPCVM to CPC-vCF

File: Disaster Recovery/SafeHaven 5 CPCVM to CLC/create-linux-dccf-clc.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {{{
-  "title": "Create Linux Protection Group-DCCF to CLC",
+  "title": "Create Linux Protection Group-CPCVM to CLC",
```
Comment (AnthonyHakim) [inline Disaster Recovery/SafeHaven 5 CPCVM to CLC/create-linux-dccf-clc.md]: Change CPCVM to CPC-vCF

File: Disaster Recovery/SafeHaven 5 CPCVM to CLC/create-linux-dccf-clc.md [hunk 3]
```diff
@@ -10,7 +10,7 @@
 This article explains how to create a Linux Protection Group.
 
 ### Requirements
-Access to the Production SRN in CenturyLink Cloud and permissions to add raw disks to it.
+Access to the Production SRN in CPCVM (DCCF) and permissions to add raw disks to it.
```
Comment (AnthonyHakim) [inline Disaster Recovery/SafeHaven 5 CPCVM to CLC/create-linux-dccf-clc.md]: I think this part of the KB refers to CLC (CenturyLink Cloud) and not the former DCCF product. I would change this to CLC.

File: Disaster Recovery/SafeHaven 5 CPCVM to CLC/create-windows-pg-dccf-clc.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {{{
-  "title": "Create Windows Protection Group, Install LRA and Start Replication- DCCF to CLC",
+  "title": "Create Windows Protection Group, Install LRA and Start Replication- CPCVM to CLC",
```
Comment (AnthonyHakim) [inline Disaster Recovery/SafeHaven 5 CPCVM to CLC/create-windows-pg-dccf-clc.md]: Change CPCVM to CPC-vCF

File: Disaster Recovery/SafeHaven 5 CPCVM to CLC/master-dccf-clc.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 {{{
-  "title": "Master Article - DCCF(Production) to CLC(DR) - SafeHaven Setup",
+  "title": "Master Article - CPCVM(Production) to CLC(DR) - SafeHaven Setup",
```
Comment (AnthonyHakim) [inline Disaster Recovery/SafeHaven 5 CPCVM to CLC/master-dccf-clc.md]: Change CPCVM to CPC-vCF

File: Disaster Recovery/SafeHaven 5 CPCVM to CLC/master-dccf-clc.md [hunk 6]
```diff
@@ -17,7 +17,7 @@ This article explains how to setup SafeHaven at a high level and use SafeHaven f
 
 1. [Gather Production Server Information](../SafeHaven 5 CLC to AWS/Gather Production Server Information.md)
 
-2. [Create Production SRN in DCCF](../SafeHaven 5 Manual to AWS/Create-Production-SRN-Manual.md)
+2. [Create Production SRN in CPCVM](../SafeHaven 5 Manual to AWS/Create-Production-SRN-Manual.md)
```
Comment (AnthonyHakim) [inline Disaster Recovery/SafeHaven 5 CPCVM to CLC/master-dccf-clc.md]: Change CPCVM to CPC-vCF

File: Disaster Recovery/SafeHaven 5 CPCVM to CLC/register-dc-dccf-clc.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 
 {{{
-  "title": "Register sites within a SafeHaven Console - DCCF to CLC",
+  "title": "Register sites within a SafeHaven Console - CPCVM to CLC",
```
Comment (AnthonyHakim) [inline Disaster Recovery/SafeHaven 5 CPCVM to CLC/register-dc-dccf-clc.md]: Change CPCVM to CPC-vCF

**Comment by jim-phillips** [comment]
@AroraAnshul Please update, based on Anthony's comments.

**Pull request merged**